src/status_im/events.cljs

```diff
@@ -36,6 +36,7 @@
     status-im.contexts.wallet.send.events
     status-im.contexts.wallet.signals
     [status-im.db :as db]
+    status-im.navigation.effects
```

Comment:
I've added this require to ensure the navigation effects are loaded. Atm the navigation effects are imported by `status-im.navigation.events` but that could change if that `status-im.navigation.events` doesn't need to depend on `status-im.navigation.effects` eventually.

Meaning that this change is here to prevent subtle bugs and it also follows the pattern used by wallet namespaces in the same file.